docs/indexing.md

```diff
@@ -142,7 +142,9 @@ Data is easily deleted from the index by the unique identifier you provided in y
 
 <!-- Tabs -->
 <div class="container">
-  <input type="radio" id="tab-link-20" name="docwriting" checked />
```

Comment:
This doc doesn't merge with the feature/docfx branch where the docs now live so this wasn't cherry picked. For doc updates now, They'll have to be done against feature/docfx for now.

Reply:
This should be fine the other file will be the one that will come on the docfx site😉